### ISSUE TITLE
`code/examples/create_bot.py` chokes at end of process when invoked outside of `code/examples`

### ISSUE THREAD

**Issue by kairstenfay** [eric-downes/deep_learning_and_the_game_of_go#1]
I just ran `./code/examples/create_bot.py` from the top level of this repo. 

The process (at one epoch) ran for many minutes before exiting with the following error:
```
Traceback (most recent call last):
  File "code/examples/create_bot.py", line 50, in <module>
    main()
  File "code/examples/create_bot.py", line 44, in main
    model_file = h5py.File("../agents/deep_bot.h5", "w")
  File "/Users/kairstenfay/miniconda3/lib/python3.6/site-packages/h5py/_hl/files.py", line 394, in __init__
    swmr=swmr)
  File "/Users/kairstenfay/miniconda3/lib/python3.6/site-packages/h5py/_hl/files.py", line 176, in make_fid
    fid = h5f.create(name, h5f.ACC_TRUNC, fapl=fapl, fcpl=fcpl)
  File "h5py/_objects.pyx", line 54, in h5py._objects.with_phil.wrapper
  File "h5py/_objects.pyx", line 55, in h5py._objects.with_phil.wrapper
  File "h5py/h5f.pyx", line 105, in h5py.h5f.create
OSError: Unable to create file (unable to open file: name = '../agents/deep_bot.h5', errno = 2, error message = 'No such file or directory', flags = 13, o_flags = 602)
```

This is because `../agents/deep_bot.h5` is hard-coded as a relative file path to the script. Ideally this file path would be absolute or the script would check if the file exists before running the expensive processes of `create_bot.py`.

**Comment by helioseven** [comment]
Absolute paths seem like tripping hazard for newcomers, I would prefer to check for existence instead (probably just relative _folder_ path, since we don't expect the .h5 file to exist in a fresh clone). I will try and put in a check for this shortly.

**Comment by helioseven** [comment]
This issue is now addressed with #5.
